index.bs

```diff
@@ -2411,6 +2415,17 @@ To <dfn>compute the channel capacity of a source</dfn> given a [=randomized resp
 
 Note: This algorithm computes the channel capacity [[CHAN]] of a q-ary symmetric channel [[Q-SC]].
 
+To <dfn>compute the channel capacity of a source with attribution scopes</dfn> given a positive integer |numTriggerStates|, a positive integer |attributionScopeLimit| and a positive integer |maxEventStates|:
+1. Let |totalStates| be |numTriggerStates| + |attributionScopeLimit| * |maxEventStates|.
+1. If |totalStates| is 1, return 0.
+1. Return log2(|totalStates|).
+
+To <dfn>compute the channel capacity of a source</dfn> given a [=randomized response output configuration=] |config|, a double |epsilon|, a possibly null positive 32-bit integer |attributionScopeLimit|, and a positive integer |maxEventStates|:
+1. Let |numTriggerStates| be the [=obtain a set of possible trigger states|number of possible trigger states=] with |config|.
+1. If |numTriggerStates| is greater than the user agent's [=max trigger-state cardinality=], return an error.
+1. If |attributionScopeLimit| is not null, return the result of [=computing the channel capacity of a source with attribution scopes=] given |numTriggerStates|, |attributionScopeLimit| and |maxEventStates|.
+1. Otherwise, return the result of [=computing the channel capacity of a source without attribution scopes=] given |config| and |epsilon|.
```

Comment:
```suggestion
1. Otherwise, return the result of [=computing the channel capacity of a source without attribution scopes=] with |config| and |epsilon|.
```